docs/index.md

```diff
@@ -0,0 +1,320 @@
+# lolex
+
+> JavaScript implementation of the timer APIs
+
+**Implements:**
+
+* [`setTimeout`](#settimeoutcallback-timeout-args)
+* [`clearTimeout`](#cleartimeoutid)
+* [`setImmediate`](#setimmediatecallback-args)
+* [`clearImmediate`](#clearimmediateid)
+* [`setInterval`](#setintervalcallback-timeout-args)
+* [`clearInterval`](#clearintervalid)
+* [`requestAnimationFrame`](#requestanimationframecallback)
+* [`cancelAnimationFrame`](#cancelanimationframeid)
+* and many more ...
+
+along with a clock instance that controls the flow of time.
+**lolex** also provides a [`Date`](#date) implementation that gets its time from the clock.
+
+**lolex** can be used to simulate passing time in automated tests and other
+situations where you want the scheduling semantics, but don't want to actually
+wait (however, from version 2.0 **lolex** supports those of you who would like to wait too).
+
+**lolex** is extracted from [Sinon.JS](https://github.com/sinonjs/sinon.js).
+
+## Installation
+
+```shell
+npm install lolex --save-dev
+```
+
+You are always free to [build it yourself](https://github.com/sinonjs/lolex/blob/53ea4d9b9e5bcff53cc7c9755dc9aa340368cf1c/package.json#L22), of course.
+
+## Usage
+
+**lolex** can be used in both Node and browser environments!
+
+In addition in browser environment **lolex** provides a [`performance`](#performancenow) implementation that gets its time from the clock.
+In Node environments **lolex** provides a [`nextTick`](#nexttick) implementation that is synchronized with the clock - and a [`process.hrtime`](#hrtimeprevtime) shim that works with the clock.
+
+If you want to use **lolex** in a browser you can use
+[the pre-built version](https://github.com/sinonjs/lolex/blob/master/lolex.js)
+available in the repo and the npm package.
+
+Using npm you only need to reference **lolex** in your `<script>` tags.
+
+```html
+<script src="./node_modules/lolex/lolex.js">
```

Comment:
unpkg?

Reply:
I took this from the main sinonJS website not sure if this makes sense there at all :smile: